tests/core/vm/contract_fixture.py

```diff
@@ -4,27 +4,25 @@
 
 # Simple Transfer Contract
 # contract code to be deployed
-simple_transfer_contract_lll_code = ['seq',
-                                        ['return',  # noqa: E127
-                                            0,
-                                            ['lll',
-                                                ['seq',
-                                                    ['mstore', 32, 1461501637330902918203684832716283019655932542976],  # noqa: E501
-                                                    ['uclamplt', ['calldataload', 0], ['mload', 32]],  # noqa: E501
-                                                    ['assert', ['call', 0, ['calldataload', 0], ['calldataload', 32], 0, 0, 0, 0]],  # noqa: E501
-                                                    'stop'],
-                                            0]]]  # noqa: E128
-
-
-# compiled byte code
-simple_transfer_contract_bytecode = b'0x61003e567401000000000000000000000000000000000000000060205260003560205181101558575060006000600060006020356000356000f1155857005b61000461003e0361000460003961000461003e036000f3'  # noqa: E501
-
-
-# address where this contract will be deployed
-simple_transfer_contract_address = generate_CREATE2_contract_address(
-    b'',
-    decode_hex(simple_transfer_contract_bytecode)
-)
+simple_transfer_contract = {
+    'lll_code': ['seq',
```

Comment:
I'm wondering if we shouldn't relegate these to their own files in a `tests/fixtures` directory somewhere along with a script to compile them and generate a JSON document with the compiled assets.

If it's possible, we could re-generate those in the CI environment to be sure we're not working with stale fixtures.  However, it'd be fine with me to commit that generated file to git in the short term as long as we have an issue to the the CI setup eventually.

Reply:
I move these to `tests/contract_fixtures` directory.(Since there's another name `tests/fixtures.py` in conflict. For now I use the name `contract_fixtures` as the fixtures put here at the moment are mainly contract code.)

And add a compile script(`contract_fixture_compile_script.py`) to generate json documents. But I don't know exactly how to automate the process of pre-run the compile script in tox. And also `Vyper` doesn't seem to support Python 3.5. So I currently commit the generated json files together.